app/services/event_disavowal/build_disavowed_event_analytics_attributes.rb

```diff
@@ -11,6 +11,7 @@ def self.call(event)
         event_type: event.event_type,
         event_created_at: event.created_at,
         event_ip: event.ip,
+        user_id: event.user_id,
```

Comment:
I missed it before, but confusingly `user_id` in the context of the `Analytics` class is actually referring to the user's UUID (a string), not the internal ID.

So we'd want:

```suggestion
        user_id: event.user.uuid,
```

(And update documentation to refer to the value being a `String`, not an `Int`)